src/components/bubble/Bubble.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+
+const Bubble = ({alignment, direction, full, children}) => {
+  let alignmentClass;
+
+  if (['left', 'right'].includes(direction)) {
```

Comment:
hm, hm. that doesn't do what I thought it does. Anyway, you are repeating strings 'left' and 'right' here and at the bottom of this file. It'd be nice to avoid that, ideas?